requirements-data_and_models.txt

```diff
@@ -18,10 +18,7 @@
 PyYAML==5.4.1
 dvc[ssh]==2.0.17
 https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_bc5cdr_md-0.4.0.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_bionlp13cg_md-0.4.0.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_craft_md-0.4.0.tar.gz
-https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_ner_jnlpba_md-0.4.0.tar.gz
+https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_core_sci_lg-0.4.0.tar.gz
```

Comment:
As you indeed say in the PR description, the only place where this `en_core_sci_lg` is used — other than the `STS_Evaluation` notebook — is for `bluesearch.database.cord_19::CORD19DatabaseCreation`.

So if has to be in the requirements, shouldn't it be in the main `requirements.txt` rather than in the `data_and_models` one? The latter is — as far as I understand — mainly for the dependencies of what we have in the dvc stuff inside `data_and_models/`.

Reply:
Good point, both `sci-lg` and `web-sm` models are now in `requirements.txt` as per commit d42ce01.

The particular syntax, including the dashes, is how the corresponding entries appear in `pip freeze`.

Reply:
Actually, for `en_core_web_sm`, shouldn't it be in `requirements-dev.txt`? It is used only in the tests, no?

Reply:
Right. It's now in `requirements-dev.txt`.